encode_utils/MetaDataRegistration/eu_register.py

```diff
@@ -338,7 +365,7 @@ def create_payloads_from_tsv(profile_id, infile):
                         val = val[:-1]
                     val = [x.strip() for x in val.split(",")]
                     # Type cast tokens if need be, i.e. to integers:
-                    val = [typecast(field_name=field, value=x, data_type=item_val_type, line_num=line_count) for x in val]
+                    val = [typecast(field_name=field, value=x, data_type=item_val_type, line_num=line_count) for x in val if x != '']
```

Comment:
Is there a good reason to explicitly check against the empty string? If not you could just do `if x`:
```suggestion
                    val = [typecast(field_name=field, value=x, data_type=item_val_type, line_num=line_count) for x in val if x]
```

Reply:
Removed empty string check